packages/cli/src/commands/addPackage.ts

```diff
@@ -0,0 +1,166 @@
+import fetch from 'node-fetch';
+import shell from 'shelljs';
+import ora from 'ora';
+import checkAndGetInstaller from '../utils/checkAndGetInstaller';
+import installPeersDependencies from '../utils/installPeersDependencies';
+import addReactNativeGestureHandlerImport from '../utils/addReactNativeGestureHandlerImport';
+import getLogger from '../utils/logger';
+import enquirer from 'enquirer';
+
+const logger = getLogger();
+
+const installPackage = async (pack: string): Promise<any> => {
+  /**
+   * Fetching package meta data
+   */
+  logger.log('Fetching package metadata ...');
+  const fetchMetaSpinner = ora('Fetching package metadata').start();
+  const [packName, version = 'latest'] = pack.split('@');
```

Comment:
no because the command is to install the packages made by the react-navigation! So they precise only it's name
@react-navigation/stack => stack only

I made it possible for them to do something like stack@5.0.0

Reply:
that's what i used the split for
for other things that are not the packages => they should npm, yarn or expo directly
our tool install peer dependencies for them

Reply:
got u , thanks